.travis.yml

```diff
@@ -16,6 +16,5 @@ script:
     - cd ..
     - mkdir _build
     - cd _build
-    - cmake -DGIT_PROTOCOL=git:// -DCDAT_BUILD_MODE=LEAN -DCDAT_BUILD_GRAPHICS=ON -DCDAT_BUILD_SCIPY=OFF ../uvcdat
-    - ctest -VV -S ../uvcdat/CMake/travis_build.cmake
-    - ctest -VV -S ../uvcdat/CMake/travis_submit.cmake
```

Comment:
Please don't change travis options. I explicitly set them for a reason. The travis has a timeout so we can only test lean mode for now. Also, I would recommend having a discussion over the email first -:)